README.md

```diff
@@ -1,144 +1,149 @@
 [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 [![doc](https://img.shields.io/badge/doc-readthedocs-blueviolet)](https://gpuopen-professionalcompute-libraries.github.io/rpp/)
-[![Build Status](https://travis-ci.com/GPUOpen-ProfessionalCompute-Libraries/rpp.svg?branch=master)](https://travis-ci.com/GPUOpen-ProfessionalCompute-Libraries/rpp)
 
-# ROCm Performance Primitives Library
+# ROCm Performance Primitives library
 
-AMD ROCm Performance Primitives (**RPP**) library is a comprehensive high-performance computer vision library for AMD processors with `HIP`/`OpenCL`/`CPU` back-ends.
-
-## Latest Release
-
-[![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/GPUOpen-ProfessionalCompute-Libraries/rpp?style=for-the-badge)](https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/releases)
-
-## Top level design
+AMD ROCm Performance Primitives (**RPP**) library is a comprehensive, high-performance computer
+vision library for AMD processors that have `HIP`, `OpenCL`, or raw `CPU` backends.
 
 <p align="center"><img width="50%" src="https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/raw/master/docs/data/rpp_structure_4.png" /></p>
 
-## Supported Functionalities and Variants
+The latest RPP release is: [![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/GPUOpen-ProfessionalCompute-Libraries/rpp?style=for-the-badge)](https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/releases)
 
-### Supported Functionalities List
+## Supported functionalities and variants
 
 <p align="center"><img width="90%" src="https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/raw/master/docs/data/supported_functionalities.png" /></p>
 
-### Supported Functionalities Samples
-
 <p align="center"><img width="90%" src="https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/raw/master/docs/data/supported_functionalities_samples.jpg" /></p>
 
 ## Documentation
 
-Run the steps below to build documentation locally.
+You can build our documentation locally using the following code:
 
-* Sphinx documentation
-```bash
-cd docs
-pip3 install -r sphinx/requirements.txt
-python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
-```
+* Sphinx
 
-* Doxygen 
-```
-doxygen .Doxyfile
-```
+  ```bash
+  cd docs
+  pip3 install -r .sphinx/requirements.txt
+  python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
+  ```
 
-## Prerequisites
+* Doxygen
+
+  ```bash
+  doxygen .Doxyfile
+  ```
 
-* **OS**
-  + Ubuntu `20.04`/`22.04`
-  + CentOS `7`
-  + RHEL `8`/`9`
-  + SLES - `15-SP4`
+## Prerequisites
 
-* [ROCm supported hardware](https://docs.amd.com/bundle/Hardware_and_Software_Reference_Guide/page/Hardware_and_Software_Support.html)
+Refer to our
+[Linux GPU and OS support](https://rocm.docs.amd.com/en/develop/about/compatibility/linux-support.html)
```

Comment:
latest branch?

Reply:
I wasn't sure if we should use the current link or the one we'll use for 6.0 (the file name is different).